### ISSUE TITLE
Placeholder text contrast incorrect

### ISSUE THREAD

**Issue by jnurthen** [adobe/react-spectrum#2759]
# 🐛 Bug Report 

Placeholder text is `--spectrum-global-color-gray-600`

it should be `--spectrum-global-color-gray-800`

## 🤔 Expected Behavior

Should meet color contrast ratios - current text does not.

## 🔦 Context

Placeholders currently fail WCAG - this mostly impacts Pickers where placeholder text is commonly used for the unselected value.

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   |
| Browser          |
| Operating System |

**Comment by jnurthen** [comment]
See https://github.com/adobe/spectrum-css/blob/b0de4e8718cf8da90447dfd03fa9a783cf6087e6/components/vars/css/globals/spectrum-colorAliases.css#L270 for spectrum-css code

**Comment by snowystinger** [comment]
Wasn't there a worry that if we did this, the placeholder would look too much like a selected option?
